### PR TITLE
[KIWI-282] - Reservation information

### PR DESCRIPTION
**Pull request by juanmnoronha**
## What does this PR do?
- Creates `qa-reservation` component
- Imports `qa-reservation`  on `qa-comentarios` component

## Screenshot
screen shot 2019-01-23 at 18 42 10

### CODE REVIEW

File: qa-comentarios/qa-comentarios.html [hunk 1]
```diff
@@ -24,6 +25,9 @@ <h2>Observações gerais</h2>
 				
 			</div>
 
+			<template	is="dom-if" if="{{hasReservation(reservation)}}">
```
Comment (gusxavier) [inline qa-comentarios/qa-comentarios.html]: additional spaces in here

File: qa-comentarios/qa-comentarios.js [hunk 2]
```diff
@@ -1,105 +1,109 @@
...
+				$(self).find('.pObservation').show();
+			});
+		},
+
+		getDate: function (timestamp) {
+			return moment(timestamp).format("DD/MM/YYYY - HH:mm:ss");
+		},
+
+		hasReservation: function (reservation) {
+			const reservationArr = reservation && Object.keys(reservation).length !== 0;
```
Comment (gusxavier) [inline qa-comentarios/qa-comentarios.js]: I think here we could do just in one line, like:

```javascript
return reservation && Object.keys(reservation).length !== 0;
```

WDYT?

File: qa-reservation/qa-reservation.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.reservation {
+    margin: 16px 0 24px;
+}
+
+    .reservation__title {
```
Comment (gusxavier) [inline qa-reservation/qa-reservation.css]: I think indentation is wrong in here. This block should be indented as the first one.

File: qa-comentarios/qa-comentarios.js (hunk 2 above)
Comment (juanmnoronha) [reply qa-comentarios/qa-comentarios.js]: NICE

File: qa-comentarios/qa-comentarios.html (hunk 1 above)
Comment (juanmnoronha) [reply qa-comentarios/qa-comentarios.html]: TKS

File: qa-reservation/qa-reservation.css (hunk 3 above)
Comment (juanmnoronha) [reply qa-reservation/qa-reservation.css]: This indentation is part of the **CSS** **architecture** concept called BEM, for better visualization of the divisions (blocks, elements and modifiers), only this!

Comment (gusxavier) [reply qa-reservation/qa-reservation.css]: Nice, didn't know about this! I'll read more about it later. :+1:

**Pull request merged**